src/parser/jobs/drg/modules/BloodOfTheDragon.js

```diff
@@ -213,20 +264,86 @@ export default class BloodOfTheDragon extends Module {
 		}))
 	}
 
+	_windowTable(window) {
+		const casts = window.nastronds.concat(window.stardivers)
+		casts.sort((a, b) => { return a.timestamp - b.timestamp })
+
+		const rows = casts.map(cast => {
+			const buffs = cast.buffs.map(id => {
+				return <StatusLink key={id} showName={false} iconSize="35px" {...getDataBy(STATUSES, 'id', id)}	/>
+			})
+
+			return <Table.Row key={cast.timestamp}>
+				<Table.Cell>{this.createTimelineButton(cast.timestamp)}</Table.Cell>
+				<Table.Cell><ActionLink {...cast.action} /></Table.Cell>
+				<Table.Cell>{buffs}</Table.Cell>
+			</Table.Row>
+		})
+
+		const buffsInDelayWindow = {}
+		let canBeDelayed = window.activeBuffs.length === 0
+		let couldBeDelayed = false
+
+		for (const id in window.timeToNextBuff) {
+			buffsInDelayWindow[id] = window.timeToNextBuff[id] >= LOTD_BUFF_DELAY_MIN && window.timeToNextBuff[id] <= LOTD_BUFF_DELAY_MAX
+			couldBeDelayed = buffsInDelayWindow[id] || couldBeDelayed
+			canBeDelayed = window.timeToNextBuff[id] >= LOTD_BUFF_DELAY_MIN && canBeDelayed
+		}
+
+		const delayBuffs = Object.keys(buffsInDelayWindow).filter(id => buffsInDelayWindow[id]).map((id, idx) => {
+			const action = getDataBy(ACTIONS, 'id', parseInt(id))
+			return <Message.Item key={idx}><Trans id={`drg.blood.delay-buff.${id}`}><ActionLink {...action} /> in {this.parser.formatDuration(window.timeToNextBuff[id])}</Trans></Message.Item>
```

Comment:
I don't think you can do `<Trans>` IDs like this - it doesn't show up in the extract, at any rate. It should be fine to just use one string though (e.g. `<Trans id="drg.blood.delay-buff">`) - since the only part that's going to be translated is the word "in", you shouldn't need 3 different versions for the 3 different buffs.

Reply:
Hm yeah I dunno why I wanted to put the ID on that one. I'm not super familiar with how the `<Trans>` tags work and I think I did something similar in a different PR that I'll have to fix as well.